lnrpc/lightning.proto

```diff
@@ -3541,6 +3541,13 @@ message Invoice {
     Note: Output only, don't specify for creating an invoice.
     */
     map<string, AMPInvoiceState> amp_invoice_state = 28;
+	
+    /*
+    The number of routing hints the invoice should include. Poor channel
+	liquidity may result in more hints being added, lack of private channels
+	in less. The maximum is 20 hints.
+    */
+    int64 hints_target = 29;
```

Comment:
hmm this commit doesn't look right...did you run `make rpc-check`?

Reply:
Errors even with the unchanged source. What is the correct way to compile the proto files? Is there a Docker image I should use?